packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # Changelog
 
+
+## 2.7.1 - Oct 18, 2021
+
+Incremental version for Teams Toolkit with multiple bugs fixes and the following updates:
+
+New Feature:
+- Support Jenkins CI/CD pipeline.
+- Support certificate-based authentication for JavaScript SDK
```

Comment:
Do we need to mention what's the SDK API and what's the additional parameter for this certificate support?